src/main/webapp/app/exam/participate/summary/exam-participation-summary.component.ts

```diff
@@ -27,9 +29,40 @@ export class ExamParticipationSummaryComponent implements OnInit {
     readonly IncludedInOverallScore = IncludedInOverallScore;
     readonly SUBMISSION_TYPE_ILLEGAL = SubmissionType.ILLEGAL;
 
+    /**
+     * If true, expects {@link studentExam} input is provided before ngOnInit and fetches StudentExamWithGradeDTO
+     * (without studentExam) from server itself.
+     * Otherwise, expects {@link studentExamWithGradeDTO} input is provided by the parent component with {@link StudentExamWithGradeDTO#studentExam} set.
+     */
+    @Input()
+    shouldFetchGradeInfoSeparately = false;
```

Comment:
please remove this!

Reply:
Addressed in https://github.com/ls1intum/Artemis/pull/4932#discussion_r901107949